core/wiki/macros/sidebar-resizer.tid

```diff
@@ -0,0 +1,551 @@
+		storyWidthConverted={{{ [function[convert.to.result],<storyWidth>,<storyWidthMetric>] }}}
+		storyMinWidthConverted={{{ [function[convert.to.result],<storyMinWidth>,<storyWidthMetric>] }}}
+		storyWidthMaxConverted={{{ [function[convert.to.result],<storyWidthMax>,<storyWidthMetric>] }}}
+		storyWidthSetTiddler={{{ [<storyWidthTiddler>!is[blank]] :else[<set.theme.metric storywidth>] }}}>
+
+		<$action-setfield $tiddler=<<storyWidthSetTiddler>> text=<<get.story-width.centralised.result>>/>
+
+	</$let>
+
+</$let>
+\end
+
+\function get.drag.diff()
+[<event-fromcatcher-posx>subtract<startPosX>]
+:map[<drag.direction.reverse>match[yes]then<currentTiddler>multiply[-1]else<currentTiddler>]
+\end
+
+\procedure sidebar-resizer-pointermove-actions()
+<%if [<get.resizer.state>is[tiddler]] %>
+	<$let
+		startPosX={{{ [<get.resizer.state>get[start-posx]] }}}
+		widgetNodeWidth={{{ [<get.resizer.state>get[widget-node-width]] }}}
+		dragDiff=<<get.drag.diff>>>
+		<%if [<tv-set-storywidth-storyright>match[yes]] %>
+			<<set-theme-throttling storyright storyRightTiddler>>
+			<<set-theme-throttling tiddlerwidth tiddlerWidthTiddler>>
+			<<set-theme-throttling storywidth storyWidthTiddler>>
+			<<set-storywidth-storyright-actions>>
+		<% endif %>
+		<%if [<tv-set-sidebarwidth>match[yes]] %>
+			<<set-theme-throttling sidebarwidth sidebarWidthTiddler>>
+			<<set-sidebarwidth-actions>>
+		<% endif %>
+		<%if [<tv-set-centralised>match[yes]] %>
+			<<set-theme-throttling storywidth storyWidthTiddler>>
+			<<set-centralised-actions>>
+		<% endif %>
+	</$let>
+<% endif %>
+\end
+
+\function get.resizer.class() tc-slider [<get.resizer.state>is[tiddler]then[tc-resizer-active]] [<resizer.class>] [<resizer.class>is[blank]then<qualify>addsuffix<currentTiddler>sha256[]addprefix[tc-slider-identified-]] +[join[ ]]
+
+\function get.active.class(class) [<class>] [<resizer.class>match[tc-main-slider]] [<eventCatcherClass>] [<get.resizer.state>is[tiddler]then[tc-resizer-active]] +[join[ ]]
+
+\function get.sidebar-resizer.pointerdown.actions()
+[<get.resizer.state>match[$:/state/sidebar/resizing]then<sidebar-resizer-pointerdown-actions>]
+:else[<sidebar-resizer-pointerdown-actions-inner>]
+\end
+
+\function get.sidebar-resizer.pointercancel.actions() [<get.resizer.state>is[tiddler]then<sidebar-resizer-pointercancel-actions>]
+
+\function get.template-right.style.left() [<sliderCondition>match[no]then[0px]] :else[[calc(2 * ]addsuffix<get.resizer.style.width>addsuffix[)]]
+
+\function get.template-left.style.flex-basis() [<sliderCondition>match[no]then[100%]] :else[[calc(]addsuffix<get.resizer.style.left>addsuffix[ - ]addsuffix<resizerWidthDivided>addsuffix[px)]]
+
+\function get.template-right.style.flex-basis() [<sliderCondition>match[no]then[0%]] :else[[calc(]addsuffix<get.template-right.width>addsuffix[ - ]addsuffix<resizerWidthDivided>addsuffix[px)]]
+
+\procedure sidebar-resizer(templateLeft:"",templateRight:"",mode:"block",eventCatcherClass:"",zIndexLeft:"")
```

Comment:
Is there a `\whitespace trim` in the code?
Are all the extra new-lines here needed? Are they there for readability or to create redundant DOM elements?
Do they create redundant DOM elements?